### PR TITLE
owners: Remove tonya11en from AWS files

### PR DESCRIPTION
**Pull request by tonya11en**
I stopped working at AWS in 2021. Removing myself from the relevant OWNERS file entries.

### CODE REVIEW

**Review by suniltheta — APPROVED** [review]

**Comment by suniltheta** [comment]
lavignes@ is also not working on Envoy. So once #29880 lands I will update the CODEOWNERS list to include my alias

**Review by kyessenov — APPROVED** [review]
Since there seems to be some continuity decided.

**Pull request merged**
